projects/angular/src/forms/datepicker/date-container.ts

```diff
@@ -175,6 +178,36 @@ export class ClrDateContainer extends ClrAbstractContainer implements AfterViewI
     );
   }
 
+  /**
+   * Return the label for the toggle button.
+   * If there's a selected date, the date is included in the label.
+   */
+  private getToggleButtonLabel(day: DayModel): string {
+    if (day) {
+      const formattedDate = this.dateIOService.toLocaleDisplayFormatString(day.toDate());
+
+      return (
```

Comment:
both cases fallback to the old toggle label so it's not breaking if anyone supplied i18n strings